include/boost/geometry/algorithms/detail/is_valid/multipolygon.hpp

```diff
@@ -265,21 +265,22 @@ class is_valid_multipolygon
         {}
 
         template <typename Polygon>
-        inline bool apply(Polygon const& polygon) const
+        inline bool operator()(Polygon const& polygon) const
         {
-            return base::apply(polygon, m_policy, m_strategy);
+            return ! base::apply(polygon, m_policy, m_strategy);
```

Comment:
I'd probably also change the name from `per_polygon` to `is_invalid_polygon` or `is_polygon_invalid`.